### ISSUE TITLE
Problems on new install

### ISSUE THREAD

**Issue by hairylarry** [ltGuillaume/MusicFolderPlayer#55]
I think this is exactly what I want.

I'm having trouble understanding the password. I don't know how to set it and  I'd rather not have it. I just want anyone to be able to play Creative Commons Jazz on the internet.

Any time I try to play a file I get X File not found error. Do I still need a $root variable in music.php or is it picking the path up from the config files?

I'm not sure I even need playlists. Just browsing folders (albums) and playing folders (albums).

Is there a better forum for newbie questions like this?

Thanks

**Comment by ltGuillaume** [comment]
> I'm having trouble understanding the password. I don't know how to set it and I'd rather not have it.

If you don't want password protection, then you can just ignore the Wiki part describing how to set up nginx.

> Any time I try to play a file I get X File not found error. Do I still need a $root variable in music.php or is it picking the path up from the config files?

You don't need to change anything in `music.php`, nor in `music.ini` if you just use the folder `library` next to `music.*` to put the music in.

> I'm not sure I even need playlists.

Well, there will always be a queue/playlist in MusicFolderPlayer, that's how it's designed. If you mean importing/exporting or loading/saving playlists, I guess you could just hide the buttons, but I don't think they're cluttering the UI.

**Comment by hairylarry** [comment]
Here's my library in a directory lister.

https://jazz.mixremix.cc/

I installed here.

https://mixremix.cc/ccjazz/music.htm

I copied music*.* from MusicFolderPlayer-master and I created a symlink, library, in the same directory linking to the CC Jazz Library.

I get all the folders and songs and can expand folders.

When hovering over a folder I see L:Open folder. If I press L I get Enter password:. That's my password question.

When I click the play button I get X File not found.

Thanks

**Comment by ltGuillaume** [comment]
The folder and/or files inside the folder your symlink points to is probably not accessible by the user Apache uses (normally `www-data`). Be sure the folder can be read by Apache's user. See https://www.makeuseof.com/you-dont-have-permission-to-access-on-this-server/

**Comment by hairylarry** [comment]
Nope, I had that right. Just to recheck I reset everything with -R. I still get the X File not found error.

I'm going to try a new install under the jazz.mixremix.cc subdomain and see if I can get anywhere that way.

I'll let you know.

Thanks

**Comment by hairylarry** [comment]
Ok, that worked. Not really sure what the problem was.

I moved my library to jazz.mixremix.cc/library so that link takes you to the directory lister.

I copied music*.* from MusicFolderPlayer-master into the home (html) directory so the player appears at jazz.mixremix.cc/music.htm

I changed permissions of the directory tree with sudo chown -R 775 *

I changed ownership of the directory tree with sudo chmod -R www-data:larry *

I'm listening now.

I may be in touch with further questions. Now that it's working correctly I understand the functionality better.

Thanks for your help.

**Comment by hairylarry** [comment]
I understand now.

L: means left click
R: means right click
same for L = and R =

Here I was typing letters and getting the password prompt

I would prefer not to get a password prompt but for now I think I will reword L: L = etc

If I copy music.defaults.ini to music.ini and modify music.ini can I then remove music.defaults.ini?

Thanks

PS the player is great and I figured out how to show album artwork.

**Comment by ltGuillaume** [comment]
> L: means left click
> R: means right click

I should have been more clear there, sorry.

So the password is because the hotkey `L` is to lock the session (for parties).

> If I copy music.defaults.ini to music.ini and modify music.ini can I then remove music.defaults.ini?

Yes. It has this name so that you could update using `git pull` without losing your own settings.

**Comment by hairylarry** [comment]
I changed L: and L = in music.ini and music.htm. Also same for right clicks.

Is it difficult to disable the lock and the L key? I don't really need them.

I haven't been able to change themes. I changed black to blue in music.ini but didn't get any appearance change.

The program is great and exactly what I want. I am adding the album artwork and making sure everything I have so far is working correctly before I add more albums. I have several large repositiores of Creative Commons jazz to add and then, of course, there are always new releases.

Thanks

**Comment by ltGuillaume** [comment]
You can simply remove the lines https://github.com/ltGuillaume/MusicFolderPlayer/blob/6423c0b0e8c4ba2d1233b58ceb4645d2b44e83d0/music.js#L1615

and https://github.com/ltGuillaume/MusicFolderPlayer/blob/6423c0b0e8c4ba2d1233b58ceb4645d2b44e83d0/music.htm#L43

**Comment by hairylarry** [comment]
Ok, I removed the lock successfully.

I am looking for a light theme instead of a dark theme. White (ish) background with black letters and blue links.

Can I write my own theme?

Would it be better to change the CSS?

Thanks so much for all your help.

**Comment by ltGuillaume** [comment]
If you remove the theme css it's already white.

**Comment by hairylarry** [comment]
Wow. There's a lot to learn here.

I added a menu at the top in music.htm and styled it white so it was easy to read against any background.

I made the suggested changes in the theme css, tweaking until I got the colors I wanted.

I left the dark theme as an option while making the blue theme the default. It goes best with my family of websites at MixRemix.

And here's how it looks.

https://jazz.mixremix.cc/music.htm

Thanks
